rogue-thi-app/data/clubs.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "club": "Studierendenvertretung .",
+    "instagram": "https://www.instagram.com/studverthi/",
+    "website": "https://studverthi.de"
+  },
+  {
+    "club": "Hochschulband .",
+    "instagram": "https://www.instagram.com/studverthi/",
+    "website": "https://studverthi.de"
+  },
+  {
+    "club": "Hochschulkino .",
+    "instagram": "https://www.instagram.com/hochschulkino.thi/",
+    "website": "https://studverthi.de/hochschulkino"
+  },
+  {
+    "club": "think e. V.",
+    "instagram": "https://www.instagram.com/think.thi/",
+    "website": "https://think-thi.de/"
+  },
+  {
+    "club": "Neuland Ingolstadt e. V.",
+    "instagram": "https://www.instagram.com/neuland_ingolstadt/",
+    "website": "https://neuland-ingolstadt.de/"
+  },
+  {
+    "club": "Our Future e. V.",
+    "instagram": "https://www.instagram.com/ourfuture_ingolstadt/",
+    "website": "https://www.ourfuturethi.de/Ne"
+  },
+  {
+    "club": "NEWEXIST e. V.",
+    "instagram": "",
```

Comment:
maybe we should set it to `null` here instead of an empty string?

Reply:
NEWEXIST has an Instagram page: https://www.instagram.com/newexist_official/ (I think you accidentally pasted it at NICE)